aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/SharedConfigDecorator.kt

```diff
@@ -73,10 +73,11 @@ class NewFromShared(runtimeConfig: RuntimeConfig) : ConfigCustomization() {
             ServiceConfig.ConfigImpl -> writable {
                 rustTemplate(
                     """
+                    /// Creates a new service config from a shared config.
```

Comment:
maybe have this link to the `service::Config` here? Could name it `${runtimeConfig.moduleName}::Config` I think